src/lib.rs

```diff
@@ -115,47 +109,43 @@ impl Eastern for Chrono {
 
 impl GoldenNumber for Chrono {
     fn get_golden_number(&self) -> u8 {
-        let golden_number: u8 = ((self.year + 1) % 19) as u8;
-        if golden_number == 0 {
-            return 19;
+        match (self.year + 1) % 19 {
+            0 => 19,
+            n => n as u8,
```

Comment:
místo if n == 0 ... tak rovnou match

Reply:
moc hezký, mě v tu chvíli nenapadlo. V tu chvíli, kdy jsem to kopíroval z pythonu